games-util/gamemode/metadata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<maintainer type="person">
+        <email>hurikhan77+bgo@gmail.com</email>
+        <name>Kai Krakow</name>
```

Comment:
Who's going to be committing for you?

Reply:
Oh, I was under the impression that `repoman` would complain when not setting this but I was unsure about it anyways. Removed it. Thanks for the hint. Still learning. :-)

Reply:
No, no. I meant you need to add someone extra who's gonna commit for you, e.g. your proxy maintainer or proxy-maint project.

Reply:
Okay, so there's no one... What's a proper way to accomplish that?

Reply:
I've added proxy-maint as this sounds like the way to go according to Gentoo docs.